src/redux&saga/types.ts

```diff
@@ -0,0 +1,28 @@
+export type IAction = {
+    type: string,
+    payload: undefined | number | string
+}
+
+export type IEpisodesAction = {
+    type: string,
+    payload: Array<string>
+}
+
+export type IEpisode = {
+    characters: Array<string>,
+    name: string,
+    episode: string,
+    url: string,
+    id: number,
+    image: 'string'
+}
+
+export type IEpisodesResponse = {
+    status: string,
+    value: IEpisode
+}
+
+// export type ICharResponse =
```

Comment:
pls remove unused code or leave a comment for what reason you left this part of code